ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -169,6 +169,15 @@ StackLayout {
         }
     }
 
+    Connections {
+        target: chatsModel.channelView
+        onActiveChannelChanged: {
+            chatsModel.messageView.hideLoadingIndicator()
+            SelectedMessage.reset();
+            chatColumn.isReply = false;
+        }
+    }
```

Comment:
This lives in `ChannelList` (not sure why to be honest) as well. For completeness-sake, I kept it here as we're removing `ChannelList` from this view. Probably want to put it somewhere else where it makes semantically sense.